libraries/adsk/adsklib/adsklib_defs.mtlx

```diff
@@ -0,0 +1,194 @@
+<?xml version="1.0"?>
+<!--
+ Copyright (c) 2021 Autodesk, Inc.
+ All rights reserved.
+ 
+ These coded instructions, statements, and computer programs contain
```

Comment:
You need to change this copyright as this is public now. Or better just remove the copyright altogether.